integration/single/test_basic_application.py

```diff
@@ -9,6 +11,10 @@ class TestBasicApplication(BaseTest):
     Basic AWS::Serverless::Application tests
     """
 
+    @pytest.fixture(autouse=True)
+    def companion_stack_outputs(self, get_serverless_application_repository_app):
+        self.sar_app_id = get_serverless_application_repository_app
```

Comment:
are we calling the `get_serverless_application_repository_app` function here or what we are doing?

Reply:
`get_serverless_application_repository_app` is a fixture and adding fixture as parameter will call that function and return its result. Its result is the ARN of the SAR App which then be used in the parameters of the template.

Reply:
thanks Mehmet for the explanation